test/index-test.js

```diff
@@ -13,104 +16,93 @@ const correctConfigWithFilterPattern = {
   destinationARN: 'arn:aws:lambda:us-moon-1:314159265358:function:testforward-test-forward',
   filterPattern: 'Test Pattern',
 };
+const createServerless = (options, service) => {
+  const serverless = new Serverless(options);
+  serverless.cli = {
+    log() {
+    },
+  };
+  new AwsProvider(serverless, options); // eslint-disable-line no-new
+  serverless.service.update(service);
+  serverless.service.setFunctionNames(options);
+  return serverless;
+};
 const constructPluginResources = (logForwarding) => {
-  const serverless = {
-    service: {
-      provider: {
-        region: 'us-moon-1',
-        stage: 'test-stage',
-      },
-      custom: {
-        logForwarding,
-      },
-      resources: {
-        Resources: {
-          TestExistingFilter: {
-            Type: 'AWS:Test:Filter',
-          },
-        },
-      },
-      functions: {
-        testFunctionOne: {
-          name: 'functionOne',
```

Comment:
Function names are meant to override deployed function name and are not required if you want to have conventional `service-stage-functionName` format.